src/main/java/ru/mambetpages/dto/PostArticleDto.java

```diff
@@ -0,0 +1,20 @@
+package ru.mambetpages.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+import java.util.List;
+
+@Data
+public class PostArticleDto {
+    @Schema(description = "Название статьи")
+    private String title;
```

Comment:
title, content and tags should not be empty or null. 
Just add @Valid to the controller's attributes and other validation annotations to the DTO's attributes.
 
see this article https://www.baeldung.com/spring-boot-bean-validation